formfyxer/lit_explorer.py

```diff
@@ -52,15 +52,15 @@
 jurisdictions = load(os.path.join(os.path.dirname(__file__), 'data', 'jurisdictions.joblib'))
 groups = load(os.path.join(os.path.dirname(__file__), 'data', 'groups.joblib'))
 clf_field_names = load(os.path.join(os.path.dirname(__file__), 'data', 'clf_field_names.joblib'))
-with open(os.path.join(os.path.dirname(__file__), 'keys', 'spot_token.txt'), 'r') as file:
-    spot_token = file.read().rstrip()
+with open(os.path.join(os.path.dirname(__file__), 'keys', 'spot_token.txt'), 'r') as in_file:
+    spot_token = in_file.read().rstrip()
 
 
 # This creates a timeout exception that can be triggered when something hangs too long. 
 
 class TimeoutException(Exception): pass
 @contextmanager
-def time_limit(seconds):
+def time_limit(seconds:int):
```

Comment:
```suggestion
def time_limit(seconds:float):
```
`Timer` seems to take a float or an int. https://docs.python.org/3/library/threading.html#threading.Timer